CHANGELOG.md

```diff
@@ -5,6 +5,6 @@ All notable changes to this project will be documented in this file.
 Extending the adopted spec, each change should have a link to its
 corresponding pull request appended.
 
-## [1.0.0] - 2021-03-18
+## [1.0.0] - 2022-03-03
 
 This is the initial release of the module, with support for <module_name_here> resources.
```

Comment:
Update the changelog file with list of featrures being developed as part of this module.

Reference

https://github.com/terraform-ibm-modules/terraform-ibm-module-template/pull/7/files

Reply:
Updated the changelog to the format at link